core-bundle/src/DependencyInjection/ContaoCoreExtension.php

```diff
@@ -95,6 +97,28 @@ public function load(array $configs, ContainerBuilder $container): void
         ;
     }
 
+    private function handleSearchIndexer(array $config, ContainerBuilder $container): void
+    {
+        $container
+            ->registerForAutoconfiguration(IndexerInterface::class)
+            ->addTag('contao.search_indexer')
+        ;
+
+        // Set the two parameters so they can be used in our legacy Config class for maximum BC
+        $container->setParameter('contao.search.default_indexer.enable', $config['search']['default_indexer']['enable']);
+        $container->setParameter('contao.search.default_indexer.enableIndexProtected', $config['search']['default_indexer']['enableIndexProtected']);
```

Comment:
```suggestion
        $container->setParameter('contao.search.default_indexer.indexProtected', $config['search']['default_indexer']['indexProtected']);
```

Reply:
Done in 29d61964184d9384111f467f69121097b0dced5e.